src/Bookings.js

```diff
@@ -1,18 +1,25 @@
-import React from "react";
+import React, { useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
   };
 
+  const [bookings, setBookings] = useState(FakeBookings);
+
+  const getBookings = () => {
+    setBookings();
+  };
+
   return (
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults bookings={bookings} />
+        {/** this no make any difference since I imported the fake data from the json file in data folder in SearchResult file */}
```

Comment:
True, in a practical sense, but it allows for reuse of the componant and testing the componant